.experimenter.yaml

```diff
@@ -158,3 +158,11 @@ wallpaper-feature:
     onboarding-sheet:
       type: boolean
       description: This property defines whether the wallpaper onboarding is shown or not
+zoom-feature:
+  description: "The configuration for the status of the zoom feature\n"
```

Comment:
Just curious, other feature don't have `\n` at the end of the description. Does that matter?

Reply:
Yeah. So, using the "yaml" style, rather than the string style, will add the `\n` at the end of the description